spec/std/process_spec.cr

```diff
@@ -2,6 +2,28 @@ require "spec"
 require "process"
 require "tempfile"
 
+private def build_and_run(code)
```

Comment:
If this is the same implementation as in `kernel_spec.cr`, it would make sense to extract to a spec helper instead of duplicating.